src/domains/idpay-app/api/idpay_onboarding_workflow/openapi.onboarding.yml.tpl

```diff
@@ -0,0 +1,380 @@
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+  '/{initiativeId}/status':
+    get:
+      tags:
+        - onboarding
+      summary: Returns the actual onboarding status
+      operationId: onboardingStatus
+      parameters:
+        - name: initiativeId
+          in: path
+          description: The initiative ID
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Check successful
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OnboardingStatusDTO'
+              example:
+                status: ACCEPTED_TC
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+        '401':
+          description: Authentication failed
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+        '404':
+          description: The requested ID was not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+        '500':
+          description: Server ERROR
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+components:
+  schemas:
+    ConsentPutDTO:
+      title: ConsentPutDTO
+      type: object
+      properties:
+        initiativeId:
+          type: string
+          description: Unique identifier of the subscribed initiative
+        pdndAccept:
+          type: boolean
+          description: Flag for PDND acceptation
+        selfDeclarationList:
+          type: array
+          items:
+            $ref: '#/components/schemas/SelfConsentDTO'
+          description: The list of accepted self-declared criteria
+    OnboardingPutDTO:
+      title: OnboardingPutDTO
+      type: object
+      properties:
+        initiativeId:
+          type: string
+          description: Unique identifier of the subscribed initiative
+    OnboardingStatusDTO:
+      title: OnboardingStatusDTO
+      type: object
+      properties:
+        status:
+          enum:
+            - ACCEPTED_TC
+            - ON_EVALUATION
+            - ONBOARDING_KO
+            - ONBOARDING_OK
+          type: string
+          description: actual status of the citizen onboarding for an initiative
+    RequiredCriteriaDTO:
+      required:
+        - listaStatoPerAnno
```

Comment:
Is this correct? Can't find any `listaStatoPerAnno` attribute in the object

Reply:
The property is now removed, it was a typo